cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+  \caption{\it Left: Viscosity profile based on the original \cite{stca06} formulation, intended for use with a temperature-dependence of viscosity based on the laterally averaged temperature. Right: Modified viscosity profile without boundary layers, intended for use with a temperature-dependence of viscosity based on an adiabatic temperature profile.}
+  \label{fig:steinberger-viscosity}
+\end{figure}
+
+In order to improve solver convergence, the material model has additional parameters that allow it to limit the viscosity variations. 
+Because of the resolution in this cookbook we limit the overall viscosity
+between $10^{20}$~Pa~s and $5 \times 10^{23}$~Pa~s and the lateral viscosity variations to 
+three orders of magnitude in both directions (for a total of six orders of
+magnitude. This allows the features of the flow field to be resolved. 
+\lstinputlisting[language=prmfile]{cookbooks/steinberger/doc/rheology.part.prm.out}
+In the Earth, we would expect higher viscosities in the lithosphere and 
+lower viscosities in plumes and near the core-mantle boundaries. 
+This type of viscosity formulation is appropriate for global convection models. 
+However, it does not approximate lithospheric deformation well. 
+The model only accounts for diffusion creep, so the 
+lithosphere has a high viscosity and forms a stagnant lid on top of the 
+sublithospheric mantle. In order to achieve more realistic subduction in a model like this, 
+one would have to either prescribe plate velocities at the surface 
+(forcing plate to subduct) or take into account plastic yielding (so that
+the lithosphere can break).
+
+\note{If the model takes too long to run, increase the minimum viscosity.}
+
+\paragraph{The projected density approximation.}
+Since our model is compressible, the most accurate way to solve the mass 
+conservation equation implemented in \aspect{} is to use the `projected density
+approximation'. This way, \aspect{} will compute the density gradients in the 
+mass conservation directly from the density field (interpolated onto the 
+finite element grid) rather than approximating it with a reference profile 
+or temperature/pressure derivatives of the density. 
+
+To use the projected density approximation, we need to specify it as the form of the equations we want to use, and we need to provide a field that the density values can be interpolated on. 
+The first part is handled in the `Formulation' section of the input file. This is where we can select 
+the projected density approximation as the formulation we want to use for the mass conservation equation. 
+The temperature equation uses the real density (rather than a reference profile) as well. 
+
+To allow for the interpolation, we create a compositional field that we call `density\_field'. 
+We assign the field the type `density', so that ASPECT knows that this is the field it should use to compute the density gradient required to solve the equations. ASPECT does not need to solve an 
```

Comment:
```suggestion
We assign the field the type `density', so that \aspect{} knows that this is the field it should use to compute the density gradient required to solve the equations. \aspect{} does not need to solve an 
```